business_requirement_deliverable_project_categ/wizard/generate_projects.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+
+
+class BrGenerateProjectsWithCategories(models.TransientModel):
+    _inherit = 'br.generate.projects'
+
+    @api.multi
+    def _prepare_project_task(self, line, project_id):
+        vals = super(BrGenerateProjectsWithCategories, self) \
+            ._prepare_project_task(line, project_id)
+        vals.update({'categ_ids': [(6, 0, [line.task_categ_id.id])]})
```

Comment:
IMHO we don't need a module just to update a dict values
Maybe you could use a parameter plus if from original module instead in order to avoid many many modules with small small features.